### PR TITLE
clarify payu collate documentation

### PR DESCRIPTION
**Pull request by aekiss**
see https://github.com/payu-org/payu/issues/275

### CODE REVIEW

**Comment by aekiss** [comment]
hold on, I'm tweaking this a bit more...

**Comment by aekiss** [comment]
ok done

File: docs/source/usage.rst [hunk 1]
```diff
@@ -257,10 +257,13 @@ collate these separated files into a single file. This is only necessary, and
 supported, for some models.
 
 For most jobs, collation is called automatically. But if you need to manually
-collate the ``K``\ th run, type the following::
+collate the ``K``\ th run (counting from zero), type the following::
```
Comment (aidanheerdegen) [inline docs/source/usage.rst:260]: Now with the proliferation of th's I'm concerned this isn't safe in a COVID-19 world ... :) ... also starts being a bit weird awkward. Can we change to something like "collate run output K" and "K-1 restart"?

**Comment by aidanheerdegen** [comment]
Typical. I ask for a PR and then quibble about it.

**Comment by aekiss** [comment]
fair point - it wasn't a pretty bit of writing. How's this?

**Comment by aidanheerdegen** [comment]
Great! Thanks.

**Pull request merged**